### ISSUE TITLE
show dependencies per package

### ISSUE THREAD

**Issue by dpastoor** [metrumresearchgroup/pkgr#11]
- [x] show dependencies
- [x] show reverse dependencies
- [x] add filter(s) for particular package(s)

## potential api examples:

```
pkgr plan --show-deps
pkgr plan --show-deps=dplyr,ggplot2

pkgr plan --show-revdeps
pkgr plan --show-revdeps
```

**Comment by dpastoor** [comment]
Closes #37 #38 as duplicate

**Comment by Dreznel** [comment]
Can this issue be closed?
